release/ReleaseNotes

```diff
@@ -3,8 +3,29 @@
 
 
 --------------------------------------------------------------------------------------------------------------
-ganga/python/Ganga
+ganga/python/GPIDev
 --------------------------------------------------------------------------------------------------------------
-* ...
+* Fix to MassStorageFile. The local area is now successfully cleaned up after files have been copied
```

Comment:
Not to mention improved whitespace, special character support for filenames and paths.